content/source/docs/enterprise/logging/index.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: "enterprise2"
+page_title: "Terraform Enterprise Logging"
+sidebar_current: "docs-enterprise2-home"
+---
+
+## Terraform Enterprise Logs
+
+This document contains information about interacting with Private Terraform Enterprise logs.
+
+### Audit Logs
+
+As of Private Terraform Enterprise release v201802-1, audit logging is available in Private Terraform Enterprise. 
+
+#### What will be logged?
+
+The audit log will be updated when any resource managed by Terraform Enterprise is changed. Read requests will be logged for resources deemed sensitive. These include:
+
+  * Authentication Tokens
+  * Configuration Versions
+  * Policy Versions
+  * OAuth Tokens
+  * SSH Keys
+  * State Versions
+  * Users
+  * Vars
```

Comment:
For consistency, let's say "variables." (The api endpoint is called `vars` but we never seem to call them vars in explanatory text.)